ci/build-project.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+UNREAL_PATH=${1}
+TEST_REPO_BRANCH=${2}
+TEST_REPO_URL=${3}
+TEST_REPO_UPORJECT_PATH=${4}
+TESTP_REPO_PATH=${5}
+GDK_HOME=${6}
+BUILD_PLATFORM=${7}
+BUILD_STATE=${8}
+BUILD_TARGET=${9}
+
+# Clone the testing project
+
+rm -rf ${TESTP_REPO_PATH}
+echo "Downloading the testing project from ${TEST_REPO_URL}"
+git clone -b "${TEST_REPO_BRANCH}" "${TEST_REPO_URL}" "${TESTP_REPO_PATH}" --depth 1
```

Comment:
Long form flags are preferable within scripts because
* they illustrate more what's going on
* they're easier to google
* you're going to get asked what they mean more often

short-form is really only a typing optimisation